vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/action/MobileAppScrollActions.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.mobileapp.action;
+
+import org.openqa.selenium.WebElement;
+import org.vividus.ui.action.IScrollActions;
+
+public class MobileAppScrollActions implements IScrollActions
+{
+    @Override
+    public void scrollIntoView(WebElement webElement, boolean alignedToTheTop)
```

Comment:
bad design, this API is not applicable to mobile, so it shouldn't be implemented at all (even empty)

Reply:
discussed to have own set of search attribute beans for each profile without moving similar beans into selenium extension module